cnf-certification-test/tnf_config.yml

```diff
@@ -49,3 +49,5 @@ servicesignorelist:
   - "hazelcast-platform-controller-manager-service"
   - "hazelcast-platform-webhook-service"
   - "new-pro-controller-manager-metrics-service"
+createdBy: "default-created-by"
```

Comment:
can u please give a more clear name 
what is the meeining of created by?

Reply:
I agree I think Created By and Partner Name might be the same thing/idea?

Reply:
Created By means who created the file - CI, QE, partner etc.
Partner Name specifies the current partner

Reply:
maybe TestsRanBy will be more meaningful than CreatedBy?

Reply:
executedBy

Reply:
if it takes little effort, change it.